README.md

```diff
@@ -120,6 +120,7 @@
 * [Arion](https://github.com/hercules-ci/arion) - Run `docker-compose` with help from Nix/NixOS.
 * [cached-nix-shell](https://github.com/xzfc/cached-nix-shell) - A `nix-shell` replacement that uses caching to open subsequent shells quickly.
 * [Cachix](https://cachix.org/) - Hosted binary cache service; free for open-source projects.
+* [Devbox](https://github.com/jetpack-io/devbox) - Instant, easy, and predictable development environments
```

Comment:
Idk about "instant", but I might recommend emphasizing "portable"  given `devcontainer.json` support.

Reply:
Thank you for reviewing. Added "portable". The "instant" should be related to the caching. That the next time the env starts up is faster in comparison with other package managers (used in the analogy of the project description).